### ISSUE TITLE
Careful with class property declarations - diverging behaviour

### ISSUE THREAD

**Issue by kungfooman** [futurGH/ts-to-jsdoc#39]
Given TS code:

```js
class Frame {
  bias: number;
  constructor() {
    this.bias = 0.001;
  }
}
class Multiframe extends Frame {
  bias: number;
}
const frame = new Frame();
const multiframe = new Multiframe();
console.log('frame.bias', frame.bias);
console.log('multiframe.bias', multiframe.bias);

```

And `tsc` compiles into this JS:

```js
"use strict";
class Frame {
    constructor() {
        this.bias = 0.001;
    }
}
class Multiframe extends Frame {
}
const frame = new Frame();
const multiframe = new Multiframe();
console.log('frame.bias', frame.bias);
console.log('multiframe.bias', multiframe.bias);

```

Output:

![image](https://github.com/futurGH/ts-to-jsdoc/assets/5236548/b15ad997-8e01-4900-afca-ea7addb963c5)

However, `ts-to-jsdoc` transpiles into:

```js
class Frame {
    bias = undefined;
    constructor() {
        this.bias = 0.001;
    }
}
/** @extends Frame */
class Multiframe extends Frame {
    bias = undefined;
}
const frame = new Frame();
const multiframe = new Multiframe();
console.log('frame.bias', frame.bias);
console.log('multiframe.bias', multiframe.bias);
```

Output:

![image](https://github.com/futurGH/ts-to-jsdoc/assets/5236548/5443e5cf-7131-4b6c-bb91-d3174a55357c)

**Comment by futurGH** [comment]
It's a combination of `useDefineForClassFields`, which is `true` by default but likely `false` in your project, producing 
```js
class Frame {
    bias;
}
```
and https://github.com/futurGH/ts-to-jsdoc/blob/64f2176cfeab0f9f6cab2ad29c9c4d9db083c598/index.ts#L230-L232

I'm not entirely sure why I had that check in the first place, going to see if it needs tweaking.

**Comment by futurGH** [comment]
Fixed in 2.0.0!
